.github/workflows/lint.yml

```diff
@@ -3,25 +3,39 @@ name: Lint
 on: [pull_request]
 
 jobs:
-  merge-conflict:
-    name: Merge Conflict
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Check merge conflict
-        run: grep "^<<<<<<< HEAD" $(git ls-files | xargs) && exit 1 || true
   misspell:
-    name: Check Spelling
+    name: Check spelling all files in commit misspell
     runs-on: ubuntu-latest
     steps:
       - name: Check out
         uses: actions/checkout@v2
       - name: Install
         run: wget -O - -q https://git.io/misspell | sh -s -- -b .
-      - name: Misspell
+      - name: misspell
         run: git ls-files --empty-directory | xargs ./misspell -error
+  pre-commit:
```

Comment:
this is great. running the same things in github than locally makes it easier to troubleshoot.